addons/packages/kpack/0.5.0/README.md

```diff
@@ -0,0 +1,565 @@
+| `kp_default_repository`          | Optional          | Docker repository used for builder images and dependencies (i.e. `gcr.io/my-project/my-repo` or `mydockerhubusername/my-repo`. Used with the [kp cli](https://github.com/vmware-tanzu/kpack-cli). |
+| `kp_default_repository_username` | Optional          | Username for `kp_default_repository` (Note: use `_json_key` for GCR)                                                                                                                              |
+| `kp_default_repository_password` | Optional          | Password for `kp_default_repository`                                                                                                                                                              |
+| `http_proxy`                     | Optional          | The HTTP proxy to use for network traffic                                                                                                                                                         |
+| `https_proxy`                    | Optional          | The HTTPS proxy to use for network traffic                                                                                                                                                        |
+| `no_proxy`                       | Optional          | A comma-separated list of hostnames, IP addresses, or IP ranges in CIDR format that should not use a proxy                                                                                        |
+
+## Usage Example
+
+### Getting started with kpack using the `kp` cli
+
+This guide assumes that [`kp`](https://github.com/vmware-tanzu/kpack-cli) has been downloaded. It can be installed using a [homebrew tap](https://github.com/vmware-tanzu/homebrew-kpack-cli), by downloading from the [github release](https://github.com/vmware-tanzu/kpack-cli/releases) or through a [docker image](https://hub.docker.com/r/kpack/kp).
+
+1. Log in to the `kp_default_repository` locally.
+
+    ```bash
+    docker login <REGISTRY-HOSTNAME> -u <REGISTRY-USER>
+    ```
+
+   > Note: The `<REGISTRY-HOSTNAME>` must be the registry prefix for its corresponding registry
+   > - For [dockerhub](https://hub.docker.com/) this should be `https://index.docker.io/v1/`. `kp` also offers a simplified way to create a dockerhub secret with a `--dockerhub` flag.
+   > - For [GCR](https://cloud.google.com/container-registry/) this should be `gcr.io`. If you use GCR then the username can be `_json_key` and the password can be the JSON credentials you get from the GCP UI (under `IAM -> Service Accounts` create an account or edit an existing one and create a key with type JSON). `kp` also offers a simplified way to create a gcr secret with a `--gcr` flag.
+
+   > Note: The `<REPOSITORY>` must be a location in the docker registry that can be written to with the credentials
```

Comment:
I don't see the <REPOSITORY> variable in the command on line 41?

Reply:
I will remove that